src/Controller/EnMarche/TonMacronController.php

```diff
@@ -34,18 +35,17 @@ public function redirectLegislativesAction(): Response
      * @Route("/pourquoivoterenmarche", name="app_ton_macron_invite")
      * @Method("GET|POST")
      */
-    public function inviteAction(Request $request): Response
+    public function inviteAction(Request $request, InvitationProcessorHandler $handler): Response
     {
         $session = $request->getSession();
-        $handler = $this->get('app.ton_macron.invitation_processor_handler');
         $invitation = $handler->start($session);
         $transition = $handler->getCurrentTransition($invitation);
 
         $form = $this->createForm(TonMacronInvitationType::class, $invitation, ['transition' => $transition]);
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
-            if ($invitationLog = $this->get('app.ton_macron.invitation_processor_handler')->process($session, $invitation)) {
+            if ($invitationLog = $handler->process($session, $invitation)) {
```

Comment:
Same here, you should use `$this->get(InvitationProcessorHandler:class)` and not inject the handler.

Reply:
same, $handler is used on the top of this method